dbms/src/Interpreters/Aggregator.h

```diff
@@ -796,6 +802,18 @@ struct AggregatedDataVariants : private boost::noncopyable
     using AggregationMethod_keys256_hash64 = AggregationMethodKeysFixed<AggregatedDataWithKeys256Hash64>;
     using AggregationMethod_serialized_hash64 = AggregationMethodSerialized<AggregatedDataWithStringKeyHash64>;
 
+    using AggregationMethod_key_int256_magic_hash
+        = AggregationMethodOneNumber<Int256, AggregatedDataWithInt256KeyMagicHash>;
```

Comment:
Any usage for this? I think in TiFlash there is no int256 column

Reply:
col decimal256 may use it